mvnw

```diff
@@ -256,4 +256,4 @@ printf %s\\n "$distributionUrl" >"$TMP_DOWNLOAD_DIR/$distributionUrlNameMain/mvn
 mv -- "$TMP_DOWNLOAD_DIR/$distributionUrlNameMain" "$MAVEN_HOME" || [ -d "$MAVEN_HOME" ] || die "fail to move MAVEN_HOME"
 
 clean || :
-exec_maven "$@"
\ No newline at end of file
+exec_maven "$@"
```

Comment:
exclude this file, it is file as-is from wrapper, it does not belong to us.

Reply:
then delete, as everything not belong to us, has no place in this world (repo).

Of course, we need to pass build.

Kindly consider real issue being in silly OS being used like windoof system, causing git issues when line ending not normalized, working together with real computers using linux or unix derivate.

- https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/
- https://editorconfig.org/

<img width="1159" height="694" alt="image" src="https://github.com/user-attachments/assets/a5a1a868-a682-4632-8792-6a1a8a6b2b86" />